### PR TITLE
AAE-9036 New identity adapter service

### PR DESCRIPTION
**Pull request by jesty**
This PR depends on https://github.com/Activiti/activiti-cloud/pull/737

It contains the new Identity Adapter Service and some acceptance tests.
The old end point is still included in Modeling Service to don't break anything in UI, it will be removed as soon as the UI will start using the new endpoint.
The new acceptance tests are using the old endpoint, they will be moved to the new endpoints when the old will be removed.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,7 +2,7 @@ RELEASE_VERSION := $(or $(shell cat VERSION), $(shell python -c "from xml.etree.
 ACTIVITI_CLOUD_FULL_CHART_CHECKOUT_DIR := .git/activiti-cloud-full-chart
 ACTIVITI_CLOUD_FULL_EXAMPLE_DIR := $(ACTIVITI_CLOUD_FULL_CHART_CHECKOUT_DIR)/charts/activiti-cloud-full-example
 ACTIVITI_CLOUD_FULL_CHART_BRANCH := dependency-activiti-cloud-application-$(RELEASE_VERSION)
-ACTIVITI_CLOUD_FULL_CHART_RELEASE_BRANCH := $(or $(ACTIVITI_CLOUD_FULL_CHART_RELEASE_BRANCH),master)
```
Comment (miguelruizdev) [inline Makefile]: let's not forget to change it back to master before merging

File: activiti-cloud-modeling/src/main/resources/application.properties [hunk 2]
```diff
@@ -1,5 +1,6 @@
 authorizations.security-constraints[0].authRoles[0]=${ACT_KEYCLOAK_MODELER_ROLE:ACTIVITI_MODELER}
 authorizations.security-constraints[0].securityCollections[0].patterns[0]=${ACT_KEYCLOAK_PATTERNS:/v1/*}
+authorizations.security-constraints[1].securityCollections[0].patterns[0]=/v1/identity/*
```
Comment (miguelruizdev) [inline activiti-cloud-modeling/src/main/resources/application.properties:3]: is this the config that we need to change once the FE stops using the old API?

File: Makefile (hunk 1 above)
Comment (jesty) [reply Makefile]: thanks

File: activiti-cloud-modeling/src/main/resources/application.properties (hunk 2 above)
Comment (jesty) [reply activiti-cloud-modeling/src/main/resources/application.properties:3]: yes, we need to remove when we stop using the old api

**Review by miguelruizdev — APPROVED** [review]

**Pull request merged**